src/tiled/invertycoordinatehelper.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * invertycoordinatehelper.h
+ * Copyright 2018, Anthony Divitto <anthonydivitto@gmail.com>
+ *
+ * This file is part of Tiled.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <QtMath>
```

Comment:
This include is superfluous.

Reply:
@bjorn I was using it to define qreal. I realize now that it's defined in qglobal.h (QtGlobal) is that a better option? Throws "does not name a type" without it.

Reply:
Makes sense. Feel free to include that one then:
```c++
#include <QtCore/qglobal.h>
```